### ISSUE TITLE
Release 4.0-beta2

### ISSUE THREAD

**Issue by borekb** [versionpress/versionpress#1373]
It's been a while since 4.0-beta and there's been some fixes merged so @candrews [proposed](https://gitter.im/versionpress/versionpress?at=5bd7514682893a2f3b5ce039) we cut a beta2 release.

- [x] Fix failing tests (see comments below)
    - [x] #1384
    - [x] #1400
- ~~Make sure VersionPress is compatible with Gutenberg – #1387~~ _this will require more effort, postponing..._
- [x] Update dependencies (there are some npm audit warnings right now) #1418
- [x] Update GitHub projects
    - [x] Rename [project no. 3](https://github.com/versionpress/versionpress/projects/3) to "4.0-beta → 4.0-beta2"
    - [x] Create a new project for post-beta2 issues. [Project no. 4](https://github.com/versionpress/versionpress/projects/4).
- ~~Create a milestone for beta2.~~ Maybe later
- [x] Write release notes #1426
- [x] Tag the version, build the release.
- [ ] Publish it 🎉

**Comment by lframnes** [comment]
Hello, in an effort to help out @candrews I ran the full test suite (though on a Mac).  The unit tests all passed and the following workflow test failed.  I am not sure if this is a new failure or introduced since the last release.  Dropping in the output below:

1) VersionPress\Tests\Workflow\CloneMergeTest::dateModifiedMergesAutomatically
Failed asserting that two strings are equal.
--- Expected
+++ Actual
@@ @@
-'2019-02-19 22:40:52
+'2019-02-19 22:40:46
 '

/opt/versionpress/tests/Workflow/CloneMergeTest.php:146

**Comment by borekb** [comment]
I think that's the one that also consistently fails for me. I didn't have time to look into it closer but if someone wanted to fix it, that would be great.

**Comment by borekb** [comment]
BTW, thank you very much for helping out, @lframnes!

**Comment by candrews** [comment]
So... when can we do a release?

**Comment by borekb** [comment]
I've updated the checklist above, will be happy to release beta2 when it's done. If someone could fix the tests, that would be appreciated.

**Comment by candrews** [comment]
Especially since we now know the VersionPress isn't compatible with the current version of Wordpress (5.1), see #1374, I think this release should be done really soon.

**Comment by pavelevap** [comment]
@lframnes: I created a separate issue for failing test `dateModifiedMergesAutomatically`, see #1382.

**Comment by candrews** [comment]
What's currently blocking the release of beta 2?

**Comment by borekb** [comment]
@candrews The biggest issue right now is that editor actions done via Gutenberg are tracked as non-admin, see #1387. We can't ship beta2 before that is fixed.

Other than that, we've made a lot of progress on tests, see #1389 and related pull requests, which I think I'll be able to wrap up later this week.

Things I could use help with is:

- Debug or (even better) fix #1387.
- Prepare a draft of release notes.

**Comment by borekb** [comment]
@candrews, FYI, we've discussed this with @pavelevap again and because supporting Gutenberg will be a bigger task, I agree with your original suggestion to just release beta2 at the current state.

As it's been a while since the last release, I need to gather what has changed and write some release notes. This will take me a day or two but there are no major blockers now, especially since tests pass again (#1400).

**Comment by candrews** [comment]
Can this release be made soon please? :angel:

**Comment by borekb** [comment]
😅 I almost thought you forgot :) Just need to find a little bit of time but OK, let me commit to a release this week.

**Comment by borekb** [comment]
I've tagged a release and published a binary [here](https://github.com/versionpress/versionpress/releases/tag/4.0-beta2) 🎉  . @pavelevap / @candrews, could you please verify that it works? I need to catch some sleep.

Release notes [here](https://github.com/versionpress/versionpress/blob/1e6087c453f48d4f917f0392c436d815aa514744/docs/content/en/release-notes/4.0-beta2.md).

**Comment by candrews** [comment]
It seems to be working well for me. I tested it with 2 sites and haven't noticed problems yet :)

**Comment by borekb** [comment]
Ok, I've updated the [release page](https://github.com/versionpress/versionpress/releases/tag/4.0-beta2), let's consider beta2 released!

**Comment by pavelevap** [comment]
Yes, everything seems fine, good work @borekb :+1: 

![image](https://user-images.githubusercontent.com/3008007/56162214-bae3c580-5fcb-11e9-871d-1c62d1bea56c.png)
